client/yarn.lock

```diff
@@ -15747,4 +15747,4 @@ zustand@^4.5.2:
 zwitch@^2.0.0:
   version "2.0.4"
   resolved "https://registry.yarnpkg.com/zwitch/-/zwitch-2.0.4.tgz#c827d4b0acb76fc3e685a4c6ec2902d51070e9d7"
-  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
+  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
```

Comment:
`yarn.lock` should not have been changed if no changes to packages were introduced. New line at the end of the default ending.

Reply:
There is no difference (hash is the same)

Reply:
I know.

Reply:
It's still here.